ui/admin/tests/acceptance/scopes/update-test.js

```diff
@@ -121,61 +118,57 @@ module('Acceptance | scopes | update', function (hooks) {
         }
       );
     });
-    await visit(urls.orgScopeEdit);
+
+    await click(`[href="${urls.orgScopeEdit}"]`);
     await click('form [type="button"]', 'Activate edit mode');
     await fillIn('[name="name"]', 'random string');
     await click('[type="submit"]');
-    assert.ok(
-      find('[role="alert"]').textContent.trim(),
-      'The request was invalid.'
-    );
-    assert.ok(
-      find('.hds-form-error__message').textContent.trim(),
-      'Name is required.'
-    );
+
+    assert.dom('[role="alert"] div').hasText('The request was invalid.');
+    assert.dom('.hds-form-error__message').hasText('Name is required.');
   });
 
   test('can discard unsaved scope changes via dialog', async function (assert) {
     assert.expect(5);
     const confirmService = this.owner.lookup('service:confirm');
     confirmService.enabled = true;
     assert.notEqual(instances.scopes.org.name, 'random string');
-    await visit(urls.orgScopeEdit);
+    await visit(urls.orgScope);
+
+    await click(`[href="${urls.orgScopeEdit}"]`);
     await click('form [type="button"]', 'Activate edit mode');
     await fillIn('[name="name"]', 'random string');
     assert.strictEqual(currentURL(), urls.orgScopeEdit);
-    try {
-      await visit(urls.globalScope);
-    } catch (e) {
-      assert.ok(find('.rose-dialog'));
-      await click('.rose-dialog-footer button:first-child');
-      assert.strictEqual(currentURL(), urls.globalScope);
-      assert.notEqual(
-        this.server.schema.scopes.where({ type: 'org' }).models[0].name,
-        'random string'
-      );
-    }
+    await click(`[href="${urls.globalScope}"]`);
+    assert.dom('.rose-dialog').exists();
+    await click('.rose-dialog-footer button:first-child', 'Click Discard');
+
+    assert.strictEqual(currentURL(), urls.globalScope);
+    assert.notEqual(
+      this.server.schema.scopes.where({ type: 'org' }).models[0].name,
```

Comment:
Asking by curiosity, I remember you used before the `this.server.db...` can we use that pattern here? does have advantatges orver using the schema?

Reply:
I also asked about this in a previous PR and we switched to using `this.server.schema`. Zed let me know that is was because schema is using the actual models for the resources while db is an internal representation of the resources.